eslint.config.js

```diff
@@ -194,6 +194,14 @@ module.exports = [
       "import/no-extraneous-dependencies": "off",
       "no-restricted-imports": ["error", { patterns: ["**/src/**"] }],
     };
+
+    if (config.processor) {
+      config.processor.meta = {
+        version: "jest/recommended",
+        name: "jest/recommended",
+      };
+    }
+
```

Comment:
Actually the previous eslint with cache was broken for the test file (exception in husky).
I'm not sure if this is correct, you might want to take a look! @JLHwung

Reply:
The `version` is supposed to be what it is in the `package.json` of `eslint-plugin-jest`, and the `name` is likely `jest/snapshot-processor`. Anyway this change should have been applied on the upstream repo. We can leave a todo note to remove that once `eslint-plugin-jest` specifies such info.